etc/behat/services/contexts/ui.xml

```diff
@@ -40,6 +40,7 @@
             <argument type="service" id="sylius.behat.page.admin.locale.create" />
             <argument type="service" id="sylius.behat.page.admin.locale.index" />
             <argument type="service" id="sylius.behat.service.accessor.notification_accessor" />
+            <argument type="service" id="sylius.behat.page.admin.locale.update" />
```

Comment:
IMO the locale pages should be together, not divided by notification accessor :broken_heart: